### DISCUSSION TITLE
Start on boot version 2

### DISCUSSION THREAD

**Discussion by MordiSacks** [postalserver/postal#1854] [Help with using Postal]
Hi,
The old wiki with the instructions on start on boot is now gone (and must likely relevant anymore),

However I did not find any mention of start on boot in the new docs

I followed the installation process on a fresh VPS and it does not start on boot
I tried to add a init.d script calling `postal start`
and that starts some of the services but not all
`postal status`  returns
```
      Name                     Command               State   Ports
------------------------------------------------------------------
postal_cron_1       /docker-entrypoint.sh post ...   Up
postal_requeuer_1   /docker-entrypoint.sh post ...   Up
postal_smtp_1       /docker-entrypoint.sh post ...   Up
postal_web_1        /docker-entrypoint.sh post ...   Up
postal_worker_1     /docker-entrypoint.sh post ...   Exit 1
```

executing `postal start` manually, brings up all the services.

can anyone assist?
Thank you

**Comment by willpower232** [comment]
If you have a look at `postal logs worker` that should confirm why the worker was not able to start.

Most of the time it will be if you are running the database and queue on the same server then it is possible for docker to start before the database/queue and then the docker container would fail to start.
